logging/bin/getlogs.py

```diff
@@ -71,12 +78,23 @@ def validate_input(dict):
 
     if dict['out-filename']: ##Check for supported file-types
 
-        if (dict['maxInt'] == 0):
-            dict['maxInt'] = 250
-
         if(type(dict['out-filename']) == list):
             dict['out-filename']= " ".join(dict['out-filename'])
 
+        if os.path.isfile(dict['out-filename']):
+            if (dict['force'] == False):
+                print("\nUser specified output file already exists. Use -f to overwrite the file.\n")
```

Comment:
Just curious as to why some print statements are started with newline and others aren't?